openbb_platform/core/openbb_core/app/model/preferences.py

```diff
@@ -3,35 +3,48 @@
 from pathlib import Path
 from typing import Literal
 
-from pydantic import BaseModel, ConfigDict, Field, PositiveInt
+from pydantic import BaseModel, ConfigDict, Field, PositiveInt, field_validator
+
+from openbb_core.env import Env
 
 
 class Preferences(BaseModel):
     """Preferences for the OpenBB platform."""
 
-    data_directory: str = str(Path.home() / "OpenBBUserData")
-    export_directory: str = str(Path.home() / "OpenBBUserData" / "exports")
-    user_styles_directory: str = str(Path.home() / "OpenBBUserData" / "styles" / "user")
     cache_directory: str = str(Path.home() / "OpenBBUserData" / "cache")
     chart_style: Literal["dark", "light"] = "dark"
+    data_directory: str = str(Path.home() / "OpenBBUserData")
+    export_directory: str = str(Path.home() / "OpenBBUserData" / "exports")
+    metadata: bool = True
+    model_config = ConfigDict(validate_assignment=True)
+    output_type: Literal[
+        "OBBject", "dataframe", "polars", "numpy", "dict", "chart", "llm"
+    ] = Field(
+        default="OBBject",
+        description="Python default output type.",
+        validate_default=True,
+    )
     plot_enable_pywry: bool = True
-    plot_pywry_width: PositiveInt = 1400
-    plot_pywry_height: PositiveInt = 762
     plot_open_export: bool = (
         False  # Whether to open plot image exports after they are created
     )
-    table_style: Literal["dark", "light"] = "dark"
+    plot_pywry_height: PositiveInt = 762
+    plot_pywry_width: PositiveInt = 1400
     request_timeout: PositiveInt = 15
-    metadata: bool = True
-    output_type: Literal["OBBject", "dataframe", "polars", "numpy", "dict", "chart"] = (
-        Field(default="OBBject", description="Python default output type.")
-    )
     show_warnings: bool = True
-
-    model_config = ConfigDict(validate_assignment=True)
+    table_style: Literal["dark", "light"] = "dark"
+    user_styles_directory: str = str(Path.home() / "OpenBBUserData" / "styles" / "user")
 
     def __repr__(self) -> str:
         """Return a string representation of the model."""
         return f"{self.__class__.__name__}\n\n" + "\n".join(
             f"{k}: {v}" for k, v in self.model_dump().items()
         )
+
+    @field_validator("output_type", mode="before")
+    @classmethod
+    def llm_mode(cls, value: str) -> str:  # pylint: disable=no-self-argument
+        """Set LLM mode."""
+        if Env().LLM_MODE:
+            return "llm"
```

Comment:
This didn't seem to work on my end for this branch:

```
# .zshrc
export OPENBB_LLM_MODE=True
```
<img width="416" alt="Screenshot 2024-04-15 at 11 45 02" src="https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/b18df077-b34d-4ce8-8881-af08ddc11493">

The output type is supposed to be `llm`, right?

Reply:
Given that other parts of the code aren't working, I suspect that the `OPENBB_LLM_MODE` detection is _probably_ broken, which is causing the rest of the functionality to fail.

Reply:
Manually trying to set the `output_type` raises the following error:

<img width="1247" alt="Screenshot 2024-04-15 at 11 57 42" src="https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/973bbeb8-7b7c-4bbe-af5d-d26ff888a158">

Is it possible somehow that things aren't being "registered" correctly somehow?

I can confirm I'm on the correct branch, and have installed using `pip install -e .`:

<img width="1174" alt="Screenshot 2024-04-15 at 11 59 01" src="https://github.com/OpenBB-finance/OpenBBTerminal/assets/14093308/78be5ed4-31ca-4f90-af80-84f0fd769e10">